src/compiler/scala/tools/nsc/tasty/TreeUnpickler.scala

```diff
@@ -133,6 +140,12 @@ class TreeUnpickler[Tasty <: TastyUniverse](
     def forkAt(start: Addr): TreeReader = new TreeReader(subReader(start, endAddr))
     def fork: TreeReader = forkAt(currentAddr)
 
+    def skipParentTree(tag: Int): Unit = {
+      if (tag == SPLITCLAUSE) ()
```

Comment:
invert the condition so you don't need the `else`?

Reply:
dotty folks assume scala 2 doesn't do if-else right and also needs extra braces everywhere.

Reply:
nice suggestion